docs/conf.py

```diff
@@ -64,7 +64,7 @@
 current_year = datetime.datetime.now().year
 # General information about the project.
 project = 'django cms'
-copyright = '2009-{}, Divio AG and contributors'.format(current_year)
+copyright = '2009-{}, djangoCMS Association and contributors'.format(current_year)
```

Comment:
Should we also change the year here? Because the djangoCMS Association exists since 2020 i think.

Reply:
should be "django CMS Association" though

Reply:
@FinalAngel yeah good point, updated.